app/code/Magento/Bundle/Test/Unit/Pricing/Price/ConfiguredPriceTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Bundle\Test\Unit\Pricing\Price;
+
+use Magento\Bundle\Pricing\Price\DiscountCalculator;
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Configuration\Item\ItemInterface;
+use Magento\Catalog\Model\Product\Configuration\Item\Option\OptionInterface;
+use Magento\Catalog\Model\Product\Option;
+use Magento\Catalog\Model\Product\Option\Type\DefaultType;
+use Magento\Bundle\Pricing\Price\ConfiguredPrice;
+use Magento\Bundle\Pricing\Adjustment\Calculator;
+use Magento\Catalog\Pricing\Price\ConfiguredPriceSelection;
+use Magento\Framework\DataObject;
+use Magento\Framework\Pricing\Amount\AmountInterface;
+use Magento\Framework\Pricing\Price\PriceInterface;
+use Magento\Framework\Pricing\PriceCurrencyInterface;
+use Magento\Framework\Pricing\PriceInfo\Base;
+use Magento\Framework\Serialize\Serializer\Json;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for \Magento\Bundle\Pricing\Price\ConfiguredPrice
+ */
+class ConfiguredPriceTest extends TestCase
+{
+    /**
+     * @var float
+     */
+    protected $basePriceValue = 100.;
+
+    /**
+     * @var MockObject
+     */
+    protected $item;
```

Comment:
Since these aren't the real objects, I would name the properties with a `Mock` suffix. Also. please mark each property as private.
```suggestion
     * @var ItemInterface|MockObject
     */
    private $itemMock;
```